frontend/src/metabase/components/ErrorPages/ErrorDiagnosticModal.tsx

```diff
@@ -90,58 +189,123 @@ export const ErrorDiagnosticModal = ({
         }}
         onSubmit={handleSubmit}
       >
-        <Form>
-          <Text>
-            {t`Select the info you want to include in the diagnostic JSON file.`}
-          </Text>
-          <Stack spacing="md" my="lg">
-            {canIncludeQueryData && (
-              <FormCheckbox name="queryResults" label={t`Query results`} />
-            )}
-            {!!errorInfo.localizedEntityName && (
-              <FormCheckbox
-                name="entityInfo"
-                label={t`${capitalize(
-                  errorInfo.localizedEntityName,
-                )} definition`}
-              />
+        {formik => (
+          <Form>
+            {enableBugReportField && (
+              <>
+                <Text py="md">
+                  {t`What were you trying to do, and what steps did you take? What was the expected result, and what happened instead?`}
+                </Text>
+                <FormTextArea name="bugReportDescription" autoFocus />
+              </>
             )}
-            <FormCheckbox
-              name="frontendErrors"
-              label={t`Browser error messages`}
-            />
-            {!!errorInfo?.logs && (
+            <Box
+              bg={enableBugReportField ? "#FAFAFB" : "transparent"}
+              p={enableBugReportField ? "lg" : "0"}
+              my="md"
+              style={{ borderRadius: enableBugReportField ? "0.5rem" : "0" }}
+            >
+              <Flex align="flex-start" gap="md">
+                <Stack spacing="xs">
+                  <Text size="lg" weight="bold">
+                    {t`Include diagnostic information`}
+                  </Text>
+                  <Text color="text-medium" w="80%">
+                    {t`This information helps ${applicationName} figure out what exactly caused the issue.`}
+                  </Text>
+                </Stack>
+                {enableBugReportField && (
+                  <Button
+                    variant="subtle"
+                    p="0"
+                    style={{ fontSize: "17px", width: "50px" }}
```

Comment:
there are [mantine style props](https://v6.mantine.dev/styles/style-props/) for these! `fz` and `w`

I don't think, though that you necessarily want to set a fixed width, it causes weird ellipsification on some screen sizes, and will probably truncate other languages strangely.


![Screen Shot 2024-11-21 at 8 11 59 AM](https://github.com/user-attachments/assets/23eca058-d32d-45a1-b27d-e6363ccc131e)